pkg/api/types.go

```diff
@@ -41,6 +42,15 @@ const (
 	// Currently the latest version.
 	PluginProtoV1_1
 
+	// PluginProtoV2_0 represents v2.0 of the agent<->scheduler plugin protocol.
+	//
+	// Changes from v1.1:
+	//
+	// * Supports fractional CU
+	//
+	// Currently the latest version.
```

Comment:
> Currently the latest version.

`PluginProtoV1_1` should be updated to say "Last used in release version v0.6.0."